### DISCUSSION TITLE
alpha-g3676d1b 对应的 meta 的哪个版本哇？

### DISCUSSION THREAD

**Discussion by gerald774** [vernesong/OpenClash#4032] [Q&A]
如何判断 openclash 里面所使用的内核对应的是 meta 的哪个版本？

**Comment by fouvromenth** [comment]
把g去了就是
[alpha-3676d1b](https://github.com/MetaCubeX/mihomo/commit/3676d1b79fe8567c9375d3e812f84eb683c8f77d)
